config/secrets.yml.enc

```diff
@@ -0,0 +1 @@
+47bUfgjz1qNOUKxDbc8eJz5r/hmQzNem5WlAmA6vb7ys9PL5GmKjhaq3YKRkv2eKeb/zNXY2/ZU6ZnRWKl4ZD4qoAjIkg1K0+SwCAwr6MSxtK5V9wVd65LATPH66AoNtkhJhFfKzpWLpoeeoAUrBJX2alb8AMm5357nXxI4Y38tbeeD2PwUjZ4nRf4LfdhdmFyR5lQx8sofUMEUZLXlMTJVRVI3yZU7uieDlPs/PNBjrqFQ7UjBt1HhIOzSqbBjBYl5oqCBq5B4fgVzY2zLMxaSyun9y2N+TvxrlYk439gdHl/GngK2IHGSUx4PEajBti6mSFBR9vHJryMo2GIqYLpp9rEAGmIdUezlDq1FIpteN4yogkdGz581alYQXFiSmMNuIj3O1L7KCDvm+nBPYwriHXeZJb3KBMsrH2NQ3AIMvyXXdFk7hgtYgDuHOsubDuZdAQKyGFPFeXvGvV5qDG0zcnIDJ7b1mRC8RNFU7TYPb16H2EtUJfO4Zd7SaSbJB5dntPXY92GZocobIB6yLpj47nHsHb5GPL2NOpDK8dDSjxglwpGU8SxKw7QtGYGhNsXb/jAwjDEWRjk+wEBO2D9OsDwdTSHuvoqswYfrTbzXY8Q==--IRLe/45OlnI3gCUs--ogeizYCd9mcc3CiHnGkBtg==
```

Comment:
I'm not 100% sure how this works, how do you encrypt /decrypt these. How do we make sure this works in staging and production?

Reply:
We need a way to distribute the encryption key for this file outside of github. `secrets.yml` and `secrets.yml.enc` are a new feature in Rails 5.1. Here's an article providing a bit of background. https://www.engineyard.com/blog/encrypted-rails-secrets-on-rails-5.1